core_backend/migrations/versions/3cdc029625e5_added_user_id_to_contentdb.py

```diff
@@ -0,0 +1,44 @@
+"""Added user_id to ContentDB
```

Comment:
After I'm done making all the changes and creating the migrations, I consolidate them so you just have the 1.
This is as easy as deleting all the new migrations, take a fresh db, upgrade to head, and ask it to create a new migration again. You'll have one file with all the changes.

Reply:
Yeah this makes sense - I'll do this once all PRs are merged and table structures finalised before merging into main. If I do it now it might create rebasing hell.